packages/scoutgame-ui/src/components/claim/components/PointsTable/UnclaimedPointsTable.tsx

```diff
@@ -19,10 +21,19 @@ export async function UnclaimedPointsTable() {
     })
   );
 
+  const [, partnerRewards = []] = await safeAwaitSSRData(
```

Comment:
I believe it would be better to let PointsTable accept an undefined value for partnerRewards. The reason is that this default value is being re-created on every render, so it will always look like a new value to PointsTable and cause it to re-render when anything else on the page changes